src/Raven.Studio/typescript/models/database/settings/discoveryUrl.ts

```diff
@@ -1,11 +1,12 @@
 ﻿/// <reference path="../../../../typings/tsd.d.ts"/>
 import jsonUtil = require("common/jsonUtil");
+import accessManager = require("common/shell/accessManager");
 
 class discoveryUrl {
     discoveryUrlName = ko.observable<string>();
     validationGroup: KnockoutValidationGroup;
 
-    static usingHttps = location.protocol === "https:";
+    static usingHttps = accessManager.default.secureServer();
```

Comment:
can we rename left part as well? This is misleading now.

Reply:
done - rename to `isSecureServer` in all relevant places